CHANGELOG.md

```diff
@@ -1,6 +1,68 @@
-## 2.7.0 (on-going)
-
-> ????/??/??
+## 2.8.0 (on-going)
+
+### Tezos Integration:
+#### Keychain:
+- Tezos key derivation is based on Edward's curve (https://github.com/satoshilabs/slips/blob/master/slip-0010.md), 
+we don't have public derivations and since Libcore does not support private ones, only derivation scheme which is supported right now is `44'/<coin_type>'/<account_index>'/<node_index>'`,
+- Hence, resulting "derived" address from a keychain is directly a hash of the public key, 
+- Used hashing algorithm is `Blake2b`,
```

Comment:
Please provide a link to what this algorithm is (probably the documentation / source code you used to implement it in our lib).